cli/commands/project_sync.go

```diff
@@ -123,19 +141,39 @@ func (cmd *ProjectSync) RunStart(ctx *cli.Context) error {
 			unisonArgs = append(unisonArgs, "-ignore", ignore)
 		}
 	}
-
 	cmd.out.Verbose.Printf("Unison Args: %s", strings.Join(unisonArgs[:], " "))
 	if err = exec.Command("unison", unisonArgs...).Start(); err != nil {
 		cmd.out.Error.Fatalf("Error starting local unison process: %v", err)
 	}
-
 	cmd.WaitForSyncInit(logFile, ctx.Int("initial-sync-timeout"), ctx.Int("initial-sync-wait"))
+}
 
-	return nil
+// For systems that have native container/volume support
+func (cmd *ProjectSync) SetupBindVolume(volumeName string) {
+
+	cmd.out.Info.Printf("Starting local bind volume: %s", volumeName)
+	exec.Command("docker", "volume", "rm", volumeName).Run()
+
+	if workingDir, err := os.Getwd(); err == nil {
+		volumeArgs := []string{
+			"volume", "create",
+			"--opt", "type=none",
+			"--opt", fmt.Sprintf("device=%s", workingDir),
+			"--opt", "o=bind",
```

Comment:
To be clear, the lack of specifying ro meant this is a read/write binding?

Reply:
Are you suggesting that we explicitly mark it `:rw`

Reply:
Just confirming its `rw`.